### ISSUE TITLE
fragment changes are "local state" and should replace history, not push

### ISSUE THREAD

**Issue by dustingetz** [hyperfiddle/hyperfiddle#211]

**Comment by dustingetz** [comment]
This is really really shitty oversight

**Comment by khardenstine** [comment]
I completely disagree, I would be surprised if I clicked a fragment and the back button didn't take me to my previous spot in the document.

**Comment by khardenstine** [comment]
Two ideas:
- Change the src button to a modal/popover that fills the entire screen, with an anchor to "pop out" the src panel ala target="_blank".
- Change the existing "src" anchor to target="_blank", so it never opens in the existing tab.

**Comment by dustingetz** [comment]
Looks like there are four swaps per click
![image](https://user-images.githubusercontent.com/124158/39485029-9a9c2914-4d45-11e8-9612-fd6523ddfbad.png)
